modules/transaction-payment/src/lib.rs

```diff
@@ -518,6 +520,7 @@ pub mod module {
 
 		/// Enable and initialize charge fee pool.
 		#[pallet::weight(<T as Config>::WeightInfo::enable_charge_fee_pool())]
+		#[transactional]
```

Comment:
there're two `#[transactional]` in this dispatch call(inner method also has transactional annotation), is that ok?

Reply:
it is ok to have nested transactional but we will still want to avoid too many nesting level if possible